auth/auth-router.js

```diff
@@ -3,9 +3,98 @@ const bcrypt = require("bcryptjs")
 const jwt = require("jsonwebtoken")
 const Customers = require("../api/customers/customers-model")
 const Vendors = require("../api/vendors/vendorss-model")
-
 const router = express.Router()
 
+//Customers
+router.post('/register', (req, res) => {
```

Comment:
we should probably make the registration paths have similar names, so this one can also be customerregister?